src/inverseRobot.js

```diff
@@ -7,7 +7,21 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const newRobot = {};
+  const values = Object.values(robot);
+  const repeatCheak = new Set(values).size === values.length;
```

Comment:
You have a working solution, but let me ask you: do you really need to store in the memory one array and the set for getting info about unique values?
Or could you just check the keys in your new object?